examples/beam_coords/job.json

```diff
@@ -9,7 +9,7 @@
     "target_y": -0.08,
     "target_z": -4.3,
     "input_files": {
-        "/work/slac/data/stdhep/egsv5_1.stdhep": "beam.stdhep"
+        "/sdf/group/hps/mc/4pt55GeV/beam/20um120nA/beam_4pt55_20um120nA_60.stdhep": "beam.stdhep"
```

Comment:
@sarahgaiser Here is an input file in an example with a path on sdf.

Reply:
outdated commit